src/views/projects/Browser.svelte

```diff
@@ -1,23 +1,32 @@
 <script lang="ts">
-  import type { BaseUrl, Project, Tree } from "@httpd-client";
+  import type { BaseUrl, Remote, Tree } from "@httpd-client";
   import type { BlobResult } from "./router";
+  import type { LoadedSourceBrowsingView } from "@app/views/projects/router";
 
   import * as utils from "@app/lib/utils";
   import { HttpdClient } from "@httpd-client";
 
   import Button from "@app/components/Button.svelte";
   import Placeholder from "@app/components/Placeholder.svelte";
+  import SourceBrowsingHeader from "./SourceBrowsingHeader.svelte";
 
   import BlobComponent from "./Blob.svelte";
   import TreeComponent from "./Tree.svelte";
 
-  export let path: string;
-  export let hash: string | undefined = undefined;
-  export let project: Project;
   export let baseUrl: BaseUrl;
-  export let tree: Tree;
-  export let revision: string | undefined;
+  export let branches: Record<string, string>;
   export let commit: string;
+  export let commitCount: number;
+  export let contributorCount: number;
+  export let defaultBranch: string;
+  export let hash: string | undefined = undefined;
+  export let path: string;
+  export let peer: string | undefined = undefined;
```

Comment:
Could you remove the default value here (and, if possible, for `hash`)? If we remove the default value, the type checker notifies us when `projects/View` stops providing the value. If that’s the case, we should also remove the property.

As a general rule of thumb you should only provide default values for properties if the component is used in multiple places and for convenience you want to omit some of the properties at some call sites.

Reply:
https://github.com/radicle-dev/radicle-interface/pull/899/commits/df135ea9872da2e237a0e756f8f1c20d52688e3e